torchsynth/config.py

```diff
@@ -62,15 +63,12 @@ def __init__(
             )
         self.reproducible = reproducible
         if self.reproducible:
-            # Currently, noise module
-            # (https://github.com/torchsynth/torchsynth/issues/255)
-            # and abstract synth parameter randomization
-            # (https://github.com/torchsynth/torchsynth/issues/253)
-            # are non-reproducible unless batch_size == BATCH_SIZE_FOR_REPRODUCIBILITY.
-            if batch_size != BATCH_SIZE_FOR_REPRODUCIBILITY:
+            # We currently only support reproducibility with batch sizes that
+            # are multiples of BASE_REPRODUCIBLE_BATCH_SIZE
+            if batch_size % BASE_REPRODUCIBLE_BATCH_SIZE != 0.0:
```

Comment:
wait why not 0 not 0.0 since these are longs or ints?

Reply:
Good catch -- will fix